docker/prognostic_run/Dockerfile

```diff
@@ -30,36 +50,35 @@ COPY workflows/post_process_run /fv3net/workflows/post_process_run
 COPY workflows/prognostic_c48_run/ /fv3net/workflows/prognostic_c48_run
 COPY external/emulation /fv3net/external/emulation
 COPY external/radiation /fv3net/external/radiation
-RUN bash install_fv3net_packages.sh \
-    /fv3net/external/vcm \
-    /fv3net/external/artifacts \
-    /fv3net/external/loaders \
-    /fv3net/external/fv3fit \
-    /fv3net/external/fv3kube \
-    /fv3net/workflows/post_process_run \
-    /fv3net/workflows/prognostic_c48_run \
-    /fv3net/external/emulation \
-    /fv3net/external/radiation
+RUN bash /fv3net/.environment-scripts/setup_environment.sh \
+    fv3net-packages \
+    ${PLATFORM} \
+    ${CLONE_PREFIX} \
+    ${INSTALL_PREFIX} \
+    ${FV3NET_DIR} \
+    ${CALLPYFORT}
+
+# Add emulation project scripts
+COPY projects/microphysics/scripts /fv3net/projects/microphysics/scripts
 
 RUN echo "ulimit -s unlimited" >> /etc/bash.bashrc && \
     mkdir /outdir && \
     chmod -R 777 /outdir
 
-
 # these are needed for "click" to work
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
 
 # Override microphysics emulation
-ENV VAR_META_PATH=/fv3net/external/emulation/microphysics_parameter_metadata.yaml
+ENV VAR_META_PATH=$FV3NET_DIR/external/emulation/microphysics_parameter_metadata.yaml
 ENV OUTPUT_FREQ_SEC=18000
 
 # Add emulation project scripts
-COPY projects/microphysics/scripts /fv3net/projects/microphysics/scripts
-RUN chmod +x /fv3net/projects/microphysics/scripts/*
-ENV PATH=/fv3net/projects/microphysics/scripts:${PATH}
+ENV PATH=$FV3NET_DIR/projects/microphysics/scripts:${PATH}
```

Comment:
Is there reason for splitting the emulation project COPY (line 62) from this ENV call?

Reply:
Thanks -- this was a side effect of 736c653.  I'll fix this depending on which direction we decide to go from https://github.com/ai2cm/fv3net/pull/2047#discussion_r984015671.